server/edusync/src/main/java/com/codestates/edusync/study/studygroup/dto/StudygroupDto.java

```diff
@@ -37,17 +60,33 @@ public static class Post {
     @Getter
     @Setter
     public static class Patch {
+        @NotNull
         private Long id;
+
```

Comment:
@NAC 를 추가하는 것이 안전해보입니다.
뜻하지 않는 오류를 만날 수 있어요.
```suggestion
    @NoArgsConstructor
    @Getter
    @Setter
    public static class Patch {
        @NotNull
        private Long id;
```

Reply:
@NAC 추가 되있습니다!!